src/components/EditDetailsModal/EditDetailsModal.jsx

```diff
@@ -1,44 +1,74 @@
-import React, { useState } from 'react';
-import languageSelector from '../../assets/languages/languageSelector';
-import './EditVenueDetailsModal.css'
-import { useSelector } from 'react-redux';
+import React, { useState } from "react";
+import languageSelector from "../../assets/languages/languageSelector";
+import "./EditVenueDetailsModal.css";
+import { useSelector } from "react-redux";
+import axios from "axios";
+import Cookies from "universal-cookie";
+import { API_URL } from "../../constants/apiUrl";
 
-const EditVenueDetailsModal = ({ field, index, onClose, editItem }) => {
-  const language = useSelector(state => state.languageReducer);
-  const [editingItem, setEditingItem] = useState('');
+const EditVenueDetailsModal = ({
+  field,
+  index,
+  onClose,
+  editItem,
+  restaurant,
+}) => {
+  const language = useSelector((state) => state.languageReducer);
+  const [editingItem, setEditingItem] = useState("");
   const [errors, setErrors] = useState({});
+  const cookies = new Cookies();
+  const jwtToken = cookies.get("token");
+  const config = {
+    headers: {
+      Authorization: `Bearer ${jwtToken}`,
+    },
+  };
 
   const handleSubmit = () => {
+    const venueId = restaurant.venues[index].id_restaurant_venue;
+    const requestBody = { field, newValue: editingItem };
+
+    axios.put(`${API_URL}/restaurant-venues/${venueId}`, requestBody, config);
```

Comment:
I consider async-await for this http request.

Reply:
resolved